### PR TITLE
Jdbc to work on 1.6

### PR DESCRIPTION
**Pull request by losipiuk**
Replacement for https://github.com/TeradataCenterForHadoop/presto-td/pull/14

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -49,11 +49,15 @@
         <cli.skip-execute>true</cli.skip-execute>
         <cli.main-class>None</cli.main-class>
 
+        <jdbc.drvier.java.version>1.6</jdbc.drvier.java.version>
```
Comment (awislowski) [inline pom.xml]: typo

File: presto-jdbc/src/main/java/com/facebook/presto/jdbc/ApacheQueryHttpClient.java [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            if (attempts > 0) {
+                sleepUninterruptibly(attempts * 100, MILLISECONDS);
+            }
+            attempts++;
+
+            HttpResponse response;
+            try {
+                response = httpAsyncClient.execute(request, null).get();
+            }
+            catch (InterruptedException e) {
```
Comment (awislowski) [inline presto-jdbc/src/main/java/com/facebook/presto/jdbc/ApacheQueryHttpClient.java]: Catching Multiple Exception would look nicer:
https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html

Comment (losipiuk) [inline presto-jdbc/src/main/java/com/facebook/presto/jdbc/ApacheQueryHttpClient.java]: It is Java 6 so we cannot do that unfortunatelly.

**Comment by losipiuk** [comment]
Closing this pull request as it is superceeded by separate prestodb/presto-jdbc-java6 project. Still not public but will by open in few days.

**Pull request closed without merging**